### DISCUSSION TITLE
About the Project Bank and Submission Instructions

### DISCUSSION THREAD

**Discussion by malakumar85** [MERLCenter/MERL-Center-public#65] [Project Bank Submissions]
## About the Project Bank
This is a catch-all for projects that MERL Center members started and/or worked on, but were never completed. In some cases, MERL Center members who submit projects are looking for collaborators. In other cases, they simply want to propose the idea for others and don't want to be involved going forward. **Note that this is an informal meeting place to discuss ideas.** Any formal work agreements must be discussed and signed outside of the MERL Center. None of the MERL Center, MERL Tech or GitHub are responsible for arrangements made on these projects.

### Scope of Projects
Generally any project that was covered in a MERL Center case study is eligible. Other projects about MERL and open source are eligible. [See here for our definition of MERL](https://github.com/MERLCenter/MERL-Center-public/blob/main/MERLdefinition.md).

### Before Submitting
- Search the project bank for similar projects
- Gather the info in the section below

### What to Include
- Description of the project
- Technical requirements (if any)
- Who worked on this, origin of the project
- Keywords / other labels to include
- Budget or other funding available (if any)
- Existing people, teams or orgs that can provide support or are currently engaged on the project
- If you want to be involved in the project going forward
- If you need credit or attribution on any work going forward

### How to Submit
- [Open this link](https://github.com/MERLCenter/MERL-Center-public/discussions) in a new tab and click on "New discussion"
- Add the category "Project Bank Submissions"
- Add the label "Project Bank"
- Add a title
- Either using your own formatting or copying the formatting from an existing Discussion, fill out the information in the section above
- Click the "Submit discussion" button

### Submission Tips
- If you want contributors, post the submission link in the MERL Center channels on the MERL Tech Slack workspace and your social media (e.g. LinkedIn, Twitter) accounts
- If a submission gets the required number of participants to restart the project, use the create issue from discussion button to launch the project.

**Create issue from discussion button**
<img width="223" alt="Screen Shot 2022-01-26 at 10 30 29 AM" src="https://user-images.githubusercontent.com/12953652/151193545-66d04994-9929-4478-91da-6f9eb84b153c.png">

## Engaging with the Project Bank
- Browse other submissions and upvote the projects you like
- If you want to get involved, post a comment and tag the person who submitted
- Project Bank links are public and do not require a GitHub login to view, so share widely!
- If you decide to work on the project outside of the MERL Center, please post contact info to let others know how to join

**Comment by malakumar85** [comment]
@andrewtpham @bjk127 - would love your feedback on here!

**Reply by nickdickinson** [reply]
What are the guidelines for the types of projects that are within scope. Are these typically MERL center guides etc. or MERL project and/or open source projects?

**Reply by malakumar85** [reply]
Thanks, @nickdickinson. I added a few lines. Anything you would add? Feel free to edit the text or comment here, if so.

**Reply by andrewtpham** [reply]
@malakumar85 The guidelines look good. One suggestion, once the project bank repo is set up or regardless of repo, when issues are created for each project submission, we can create a github project board to make it easy to view and track the progress of each submission on how they are progressing. Happy to set that up when the project bank is up!

**Reply by malakumar85** [reply]
Good idea! We use project boards a lot internally. I set up one here: https://github.com/MERLCenter/MERL-Center-public/projects/1. Feel free to modify however you see fit. I'm not sure if there's an Action to push from Discussions to a Project Board like there is with Issues. Do you know?

**Reply by andrewtpham** [reply]
Nice, I'll check if there's anything to configure on the board. Here's the workflow I'm thinking 🤔:
1. A thread is created in the `Project Bank` discussions tab.
2. As it materializes and the submission guidelines are met, an `Issue` is created from the discussion.
3. The issue based on `label = Project Bank` can be added to the [project board](https://github.com/MERLTech/MERL-Center-public/projects/1).
4. The issue progresses through the board until its published

**Reply by malakumar85** [reply]
Thanks, @andrewtpham. Yeah, I have included in the instructions for project bank submitters to manually push the button to create an `Issue` once they requisite number of people have agreed to participate. Can you set up the Action to push the `Issue` to the project board?

**Comment by nickdickinson** [comment]
Looks good. Maybe we don't per se need "more info coming". I'd like to post about https://github.com/WASHNote/jmpwashdata as a test case for the project bank. I will be updating it to add more docs and then post it here. I think probably this would fall under the research / learning component and it is of course also evaluation/monitoring data just repackaged into a R package.
